app/src/main/java/bean/pwr/imskamieskiego/MapActivity.java

```diff
@@ -355,5 +355,42 @@ public boolean onMenuItemClick(MenuItem item) {
         });
     }
 
+    @Override
+    public void setChangeFloorButtonCoords(int barHeight, int resumeCounter) {
+        float margin = 20;
+        float translationY = barHeight;
+        changeFloorButtonNewY = translationY + margin;
+
+        if (resumeCounter > 1)
```

Comment:
Co to jest resumeCounter? Z dalszej analizy kody wynika, że to właściwie zawsze będzie większe niż 1. Czy to nie powinna być wartość logiczna?